bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: expose GCP compute instance type configuration

## Description

Allow configurability of compute instance type for the data plane clusters that are created in GCP.
Related to https://issues.redhat.com/browse/MGDSTRM-9362.

This PR is about introducing the configurability.
There will be other PRs about making use of it.

Merging this PR should be safe because although the flag is introduced it is not being used at the moment.
This will also require template parameter changes in stage and production before enabling GCP functionality.

## Verification Steps

Verify that kas fleet manager runs
Verify that the new added flag appears as part of kas fleet manager binary cli flags. Verify that if you change the value the data plane cluster config changes.
Verify that fleet manager is able to run from template



## Checklist (Definition of Done)
